orca-core/src/test/groovy/com/netflix/spinnaker/orca/locks/RedisLockManagerSpec.groovy

```diff
@@ -0,0 +1,186 @@
+package com.netflix.spinnaker.orca.locks
+
+import com.netflix.spinnaker.kork.jedis.EmbeddedRedis
+import redis.clients.jedis.Jedis
+import redis.clients.util.Pool
+import spock.lang.Shared
+import spock.lang.Specification
+import spock.lang.Subject
+
+class RedisLockManagerSpec extends Specification {
```

Comment:
If we're going to migrate locking to dyno at some stage we could make an abstract class with all the tests in and have a Redis impl.